lib/carin_for_blue_button_test_kit/docs/carin_for_blue_button_v2.0.0_client_suite_description.md

```diff
@@ -0,0 +1,76 @@
+The CARIN for Blue Button v2.0.0 Client test suite verifies the conformance of client systems to the STU2 HL7® FHIR®
+[CARIN for Blue Button® Implementation Guide](http://hl7.org/fhir/us/carin-bb/STU2). These tests validate that that a
+FHIR Client can play the role of a CARIN IG for Blue Button® Requestor and retrieve and use data defined by the CARIN for
+Blue Button® IG.
+
+## Scope
+
+These tests are a **DRAFT** intended to allow implementers to perform
+preliminary checks of their systems against the requirements stated for CARIN IG for Blue Button client actors
+and [provide feedback](https://github.com/inferno-framework/carin-for-blue-button-test-kit/issues)
+on the tests. Future versions of these tests may verify other
+requirements and may change the test verification logic.
+
+## Test Methodology
+
+For these tests Inferno simulates a FHIR server playing the role of a 
+[CARIN IG for Blue Button® Responder](https://hl7.org/fhir/us/carin-bb/STU2/Conformance_Requirements.html#actors). Inferno will wait for
+the client under test to request a resource for each of the CARIN for Blue Button profiles, which include the following:
```

Comment:
I'd like to clarify this slightly having read over it again, adding that Inferno has a specific patient and set of instances that clients are expected to retrieve:

>For these tests Inferno simulates a FHIR server playing the role of a [CARIN IG for Blue Button® Responder](https://hl7.org/fhir/us/carin-bb/STU2/Conformance_Requirements.html#actors). Inferno's simulated server contains a patient (resource id 888) that includes linked examples of all CARIN IG for Blue Button profiles will all required and must support elements populated: ... [profile list]

> During execution, Inferno will wait for the client under test to issue requests and will respond to them with the requested data. Inferno will then evaluate the requests is aggregate to verify that they demonstrate that the client: [list of specific checks]

I think that's cleaner and will make it easier for testers to understand what's going on with the tests.